ports/nrf/mpconfigport.h

```diff
@@ -157,6 +157,8 @@
 #define MICROPY_MODULE_WEAK_LINKS   (1)
 #define MICROPY_CAN_OVERRIDE_BUILTINS (1)
 #define MICROPY_USE_INTERNAL_ERRNO  (1)
+#define MICROPY_ENABLE_SCHEDULER    (1)
+#define MICROPY_SCHEDULER_STATIC_NODES (1)
```

Comment:
This unconditionally enables the scheduler for all nrf boards, which might not be a good idea because some very small boards are tight on flash.  I suggest wrapping this in `#if MICROPY_HW_USB_CDC_1200BPS_TOUCH`.  (If needed we can enable it unconditionally later on.)

Reply:
I can move them to the `mpconfigboard.h` file of the NANO 33 board ?

Reply:
No I think it's better to keep them here, otherwise each board that enables this touch 1200 needs to know to also enable these features (and probably the scheduler will be enabled unconditionally anyway in the near future).